docs/source/user/versions.rst

```diff
@@ -0,0 +1,35 @@
+
+Versions Supported in Bioconda
+==============================
+Bioconda only supports some versions of conda packages.
+Here is an incomplete list of the dependencies that might be relevant to your
+conda environment.
+
+Operating Systems
+-----------------
+Bioconda only supports 64-bit Linux and Mac OS
+
+Python
+------
+Bioconda only supports python 2.7, 3.6 and 3.7. The community is currently working on adding support for
+python 3.8.
+
+(The exception to this is Bioconda packages which declare `noarch: python` and only depend on
+such packages - those packages can be installed in an environment with any version of python
+they say they can support.
+However many python packages in Bioconda depend on other Bioconda packages with architecture specific
+builds, such as `pysam`, and so do not meet this criteria.)
+
+Unsupported versions
```

Comment:
@LiterallyUniqueLogin, can you help me understand this section?
Why do we need to distinguish between smaller or bigger software? Whether it should be added to Bioconda shouldn't be a matter of size usually but can be decided in the usual contribution process, I think.

I kind of feel we don't need this "Unsupported versions" section -- but I maybe/probably just didn't get the gist of what you were getting at.

Reply:
I think there should be some link to the contributing page and the gitter so that someone who wants a new version supported has a path forward instead of feeling stuck. Is that all I should have done? I can make that change.

Once we provide a link, I was worried that you might get people who fundamentally misunderstand the amount of work required by and likelihood of such a change which could be frustrating (e.g. people who post a PR to upload a single package with a python 3.5 dependency without understanding the issues with their dependencies, or someone saying "Hey, I'd like Bioconda to support Windows. Can you guys make that happen?"). So I tried to anticipate some of those issues. But maybe instead of trying to anticipate that here we could just let people go to the gitter and get feedback there.

Thoughts?

Reply:
I see, now I understand. Thanks for the explanation!

> Thoughts?

Not yet ^^. I'll have to think about how to phrase that. Probably not something I'll do too too soon, but I've made a note to remind me!